services/deployment-agent/docker/boot.sh

```diff
@@ -35,11 +36,9 @@ fi
 # RUNNING application ----------------------------------------
 if [[ ${SC_BOOT_MODE} == "debug" ]]
```

Comment:
this is kind of mixed up. There are two things completely orthogonal
 a) log level: this corresponds to the level used by ``logging``
 b) boot mode: this is how the app is boot and is given by ``SC_BOOT_MODE``: 
         -  ``default``: shall be ``simcore-service-deployment-agent --config $APP_CONFIG --loglevel=$LOG_LEVEL``
         - ``debug-ptvsd``: shall boot as ``python3 -m ptsvsd ...`` (note that this mode requires in the entrypoint to ``python3 -m pip install ptvsd`` while starting as root as well.

Reply:
good catch! actually I'm not using loglevel there but in the config file.
cleaned up.